Doc/library/inspect.rst

```diff
@@ -667,7 +667,7 @@ function.
       ``callable`` specifically (``callable.__wrapped__`` will not be used to
       unwrap decorated callables.)
 
-   .. versionadded:: 3.10
+   .. versionchanged:: 3.10
```

Comment:
If you're using `.. versionchanged::` here, you should use `.. versionchanged::` here as well:

https://github.com/python/cpython/blob/e0449b9a7fffc0c0eed806bf4cbb8f1f65397bbb/Doc/library/inspect.rst?plain=1#L665-L668

Only the parameter was added in Python 3.5; `inspect.signature` has existed since Python 3.3.